pkg/util/interpreter/rule.go

```diff
@@ -47,7 +47,7 @@ func (r *retentionRule) SetScript(c *configv1alpha1.ResourceInterpreterCustomiza
 	c.Spec.Customizations.Retention.LuaScript = script
 }
 
-func (r *retentionRule) Run(interpreter *configurableinterpreter.ConfigurableInterpreter, args RuleArgs) *RuleResult {
+func (r *retentionRule) Run(interpreter framework.Interpreter, args RuleArgs) *RuleResult {
```

Comment:
Because it only accept `configurableinterpreter.ConfigurableInterpreter` here, so I don't think we shall change this argument.

Reply:
ok